tests/phpt/handler_should_increase_memory_limit_by_configured_amount_on_oom.phpt

```diff
@@ -5,24 +5,24 @@ Bugsnag\Handler should increase the memory limit by the configured amount when a
 $client = require __DIR__ . '/_prelude.php';
 $client->setMemoryLimitIncrease(1024 * 1024 * 10);
 
-ini_set('memory_limit', '2M');
+Bugsnag\Handler::register($client);
```

Comment:
This test could run OOM _before_ installing the handler, so I've moved this to before we reduce the memory limit and bumped the limit a bit